app/components/FoodSection.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { View, Text, StyleSheet, ScrollView } from "react-native";
+import { colors, getStyles } from "../constants/styles/global";
+
+// Will Change based off data model
+interface FoodItem {
```

Comment:
Checkout @johnmck95's pull request for the type.ts file (https://github.com/SENG480a-NutriDine/mobile/pull/31/files#diff-a78b65a9a74e4f0240a068d4c67cd1a436e1dbc0d923dbd307f720e76b987216R1). We will be using the data format defined in this file. For instance, the type.ts file does not have a price attribute for FoodItem.

Reply:
Yes I will update that, that was just temporary until we got the firebase stuff set up